### DISCUSSION TITLE
Why is the command input incomplete?

### DISCUSSION THREAD

**Discussion by lancersky** [zephyrproject-rtos/zephyr#48803] [Q&A]
**Describe the bug**
Hi, When I enter the following command,
```
ot dataset set active 0e080000000000000000000300000b35060004001fffe00208dead00beef00cafe0708fddead00beef00000510df6f63a510f00f013caa1065ade5777e030d474c2d4f70656e54687265616401028cb50410237c2a55e2f2aa8de05634dc7825ef100c0402a0f7f8
```
It will return exception as follow,
```
uart:~$ ot dataset set active USBS0000000300000b35060004001fffe00208dead00beef00
caUSBS
Error 7: InvalidArgs
```
It look like the input buffer size of SHELL to be limit. How can I adjust this limit?

Please also mention any information which could help others to understand
the problem you're facing:
 - What target platform are you using?
 - What have you tried to diagnose or workaround this issue?
 - ...

**To Reproduce**
1. mkdir build; cd build
2. cmake -DBOARD=board\_xyz
3. make

**Expected behavior**

**Impact**
What impact does this issue have on your progress (e.g., annoyance, showstopper)

**Logs and console output**
If applicable, add console logs or other types of debug information
e.g Wireshark capture or Logic analyzer capture (upload in zip archive).
copy-and-paste text and put a code fence (\`\`\`) before and after, to help
explain the issue. (if unable to obtain text log, add a screenshot)

**Environment (please complete the following information):**
 - OS: (e.g. Linux, MacOS, Windows)
 - Toolchain (e.g Zephyr SDK, ...)
 - Commit SHA or Version used
```
manifest     nrf                          HEAD                                     N/A
zephyr       zephyr                       v3.0.99-ncs1                             https://github.com/nrfconnect/sdk-zephyr
mcuboot      bootloader/mcuboot           v1.9.99-ncs1                             https://github.com/nrfconnect/sdk-mcuboot
mbedtls-nrf  mbedtls                      v3.0.0-ncs2                              https://github.com/nrfconnect/sdk-mbedtls
nrfxlib      nrfxlib                      v2.0.1                                   https://github.com/nrfconnect/sdk-nrfxlib
```
**Additional context**
Add any other context that could be relevant to your issue, such as pin setting,
target configuration, ...

**Comment by yashi** [comment]
how about [SHELL_CMD_BUFF_SIZE](https://docs.zephyrproject.org/latest/kconfig.html#CONFIG_SHELL_CMD_BUFF_SIZE)?
